spec/registry.md

```diff
@@ -55,23 +55,13 @@ where `resolvedSelector` is the _resolved value_ of a _selector_
 and `keys` is a list of strings,
 the `:string` selector function performs as described below.
 
-1. Let `compare` be the string value of `resolvedSelector`.
+1. Let `compare` be the string value of `resolvedSelector` in Unicode Normalization Form C.
```

Comment:
Since we introduce is here, let's be complete

```suggestion
1. Let `compare` be the string value of `resolvedSelector`
   in Unicode Normalization Form C (NFC) [\[UAX#15\]](https://www.unicode.org/reports/tr15)
```